Sources/PaymentButtons/PayPalCreditButton.swift

```diff
@@ -101,13 +98,21 @@ public extension PayPalCreditButton {
 struct PayPalCreditButtonView: View {
 
     var body: some View {
+
         PayPalCreditButton.Representable()
     }
 }
 
 struct PayPalCreditButtonView_Preview: PreviewProvider {
 
     static var previews: some View {
-        PayPalCreditButtonView()
+        PayPalCreditButtonWrapper()
```

Comment:
Same question here on reverting this change and using `PayPalCreditButtonView` vs creating a duplicate